docs/repositories-configure/integrations/bitbucket-integration.md

```diff
@@ -6,7 +6,7 @@ description: Enable the Bitbucket integration to have pull request status, comme
 
 The Bitbucket integration incorporates Codacy on your existing Git provider workflows by reporting issues and the analysis status directly on your pull requests.
 
-When the integration is enabled, you can also create pull request comments and Jira issues directly from Codacy when [browsing the existing issues](../../repositories/issues.md) on the repository:
+When this integration is enabled, you can also create pull request comments directly from Codacy when browsing the existing repository issues on the [commit issue tabs](../../repositories/commits.md#issues-tabs), [pull request issue tabs](../../repositories/pull-requests.md#issues-tabs), or the [Security monitor](../../repositories/security-monitor.md):
```

Comment:
<img width="253" alt="image" src="https://github.com/codacy/docs/assets/83641264/328e72a8-9d69-4e23-b7cb-5ee46354f8fd">
I'm not sure we still allow this. i think this action was removed, can you confirm @joanasteodoro?

Reply:
@claudiacodacy While we removed it on the React single issue view, we still use the old one on other pages (the ones listed in the diff):
* [Commit, single issue](https://app.codacy.com/gh/troubleshoot-codacy/sample-programs/commit?bid=35883368&cid=758394454)
* [PR, single issue](https://app.codacy.com/gh/troubleshoot-codacy/sample-programs/pullRequest?prid=12182430)
* [Security monitor](https://app.codacy.com/gh/troubleshoot-codacy/sample-programs/security#) (e.g. click InputValidation)

Reply:
okok! makes sense! thank you 🙇